### PR TITLE
housekeeping + addr DOC-2579

### PR DESCRIPTION
**Pull request by aimurphy**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/7_Connectors/connect-to-harness-container-image-registry-using-docker-connector.md [hunk 1]
```diff
@@ -87,6 +84,12 @@ Select or enter the following options:
 
 Click **Continue**.
 
+:::tip
+
+By default, Harness uses anonymous access to [Harness DockerHub](https://hub.docker.com/u/harness) to pull images. If you experience rate limiting issues when pulling images anonymously, select **Username and Password** for **Authentication** and provide login details.
+
+:::
+
```
Comment (aimurphy) [inline docs/platform/7_Connectors/connect-to-harness-container-image-registry-using-docker-connector.md:92]: @ubekishov This is what I added for the rate limiting issue

Comment (ubekishov) [reply docs/platform/7_Connectors/connect-to-harness-container-image-registry-using-docker-connector.md:92]: Looks great

File: docs/platform/7_Connectors/connect-to-harness-container-image-registry-using-docker-connector.md [hunk 2]
```diff
@@ -39,46 +38,44 @@ If you choose to override the `harnessImageConnector`, you may also avoid trigge
...
    ![](./static/connect-to-harness-container-image-registry-using-docker-connector-47.png)
 
-3. In **Name**, enter a name for this connector.  
-   Harness automatically generates the corresponding ID ([entity identifier](../20_References/entity-identifier-reference.md)).  
-   If you want to override the Account-level Connector, modify the ID and set it to `harnessImage`. You must use the Id `harnessImage`.  
+3. In **Name**, enter a name for this connector.
+   Harness automatically generates the corresponding ID ([entity identifier](../20_References/entity-identifier-reference.md)).
+   If you want to override the Account-level Connector, modify the ID and set it to `harnessImage`. You must use the Id `harnessImage`.
    Harness gives precedence to the Connector with the `harnessImage` identifier, and uses it to pull from the Harness Container Image Registry, as opposed to pulling from DockerHub directly.
```
Comment (michaelcretzman) [inline docs/platform/7_Connectors/connect-to-harness-container-image-registry-using-docker-connector.md:72]:
```suggestion
3. In **Name**, enter a name for this connector.
   Harness automatically generates the corresponding Id ([entity identifier](../20_References/entity-identifier-reference.md)).
   If you want to override the account-level connector, modify the Id and set it to `harnessImage`. You must use the Id `harnessImage`.
   Harness gives precedence to the connector with the `harnessImage` identifier, and uses it to pull from the Harness Container Image Registry, as opposed to pulling from DockerHub directly.
```

Comment (michaelcretzman) [reply docs/platform/7_Connectors/connect-to-harness-container-image-registry-using-docker-connector.md:72]: we use Id not ID. Id stands for Identifier. ID stands for Identity Document.
We also lowercase Harness entities like connector now. In the past we capitalized them, but this has changed. Lots of the old way are still in the docs, tho.

**Review by michaelcretzman — DISMISSED** [review]
approved with one suggestion.

**Pull request merged**